query-helpers/acs.js

```diff
@@ -86,67 +86,67 @@ const acsSQL = (ids, isPrevious = false) => `
       WHEN base_sum IS NULL THEN 0
       ELSE sum / base_sum
     END AS percent,
-    --- percent_m ---
+    --- percent_margin_of_error ---
     CASE
-      WHEN NOT ${isAggregate(ids)} THEN percent_m / 100
+      WHEN NOT ${isAggregate(ids)} THEN "percentMarginOfError" / 100
       WHEN base_sum = 0 THEN 0
       WHEN base_sum IS NULL THEN 0
-      WHEN POWER(m, 2) - POWER(sum / base_sum, 2) * POWER(base_m, 2) < 0
-        THEN (1 / base_sum) * SQRT(POWER(m, 2) + POWER(sum / base_sum, 2) * POWER(base_m, 2))
-      ELSE (1 / base_sum) * SQRT(POWER(m, 2) - POWER(sum / base_sum, 2) * POWER(base_m, 2))
-    END AS percent_m,
-    --- is_reliable ---
+      WHEN POWER("marginOfError", 2) - POWER(sum / base_sum, 2) * POWER(base_margin_of_error, 2) < 0
+        THEN (1 / base_sum) * SQRT(POWER("marginOfError", 2) + POWER(sum / base_sum, 2) * POWER(base_margin_of_error, 2))
+      ELSE (1 / base_sum) * SQRT(POWER("marginOfError", 2) - POWER(sum / base_sum, 2) * POWER(base_margin_of_error, 2))
+    END AS "percentMarginOfError",
+    --- isReliable ---
     CASE
-      WHEN cv < 20 THEN true
+      WHEN "correlationCoefficient" < 20 THEN true
       ELSE false
-    END AS is_reliable
+    END AS "isReliable"
```

Comment:
Do we need the quotations here?

Reply:
Yep, it won't keep the capital letters without them (lost an hour or two to that little bit of sql trivia).

Reply:
aw. Gotcha thanks!